Dockerfiles/ci/example-metadata-without-alm-annotations/0.2.33/erynis-operator.clusterserviceversion.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
     containerImage: registry.connect.redhat.com/elos/erynis-operator@sha256:330e05580bc3a4eaf8392d54aeda588079603c086cd729ada598e60c956161c5
     createdAt: '2021-02-23'
     description: Erynis Monitoring Stack based on Thanos and Grafana
-    operators.operatorframework.io/builder: operator-sdk-v1.3.0
```

Comment:
why do we need to change the example ?

Reply:
I felt that the examples should also be up-to-date.  I'm happy to change it back if you don't think it's necessary.

Reply:
They are just examples for tests in CI. And even "outdated examples make sense in CI"

Reply:
Fair point.  Updated

Reply:
I still can see the change :-(